src/ts/playerutils.ts

```diff
@@ -59,6 +59,29 @@ export namespace PlayerUtils {
     return player.getSeekableRange() && player.getSeekableRange().start || defaultValue;
   }
 
+  /**
+   * Returns seekable time range.
+   * For live streams it will calculate timestamp timerange if tweak
+   * `enable_seek_for_vod` is not present
+   */
+  export function getSeekableRange(player: PlayerAPI): TimeRange {
+    let {start, end} = player.getSeekableRange();
+
+    // If it is VOD or we have `enable_seek_for_vod` enabled
+    if (!player.isLive() || end > -1 && start > -1) {
+      return {start, end};
+    }
+
+    const currentTimeshift = -player.getTimeShift();
+    const maxTimeshift = -player.getMaxTimeShift();
+    const currentTime = player.getCurrentTime();
+
+    end = currentTime + (currentTimeshift);
+    start = currentTime - (maxTimeshift - currentTimeshift);
+
+    return {start, end};
+  }
+
```

Comment:
I don't fully get why this is needed 🤔 Shouldn't the player `player.getSeekablerange()` return what we need 🤔

Reply:
Interestingly enough, it will return `start -1 and end -1` for live streams, unless we have `enable_seek_for_live` enabled. So as a fallback we need to make sure to get these values somehow.